backend/src/main/java/moaon/backend/global/config/WebConfig.java

```diff
@@ -10,14 +10,15 @@ public class WebConfig implements WebMvcConfigurer {
     @Override
     public void addCorsMappings(CorsRegistry registry) {
         registry.addMapping("/projects/**")
```

Comment:
```suggestion
        registry.addMapping("/**")
```

이전에 CORS 설정을 방어적으로 하기 위해 projcets 경로만 허용을 했는데,
이제 articels 도메인도 추가가 되어서 **모든 경로를 허용**하는 설정으로 변경해야 할 것 같습니다!